src/GameData.ts

```diff
@@ -123,6 +154,10 @@ export default class GameData {
     public onLookup(message: Discord.Message, isAdmin: boolean, command: string, args: string[]) {
         const supportedTypes = ["item", "perk", "rune", "champion", "champ"];
 
+        if (supportedTypes.includes(command)) {
+            args.unshift(command);
+        }
+
         if (args.length === 0) {
             const response = `Usage: !${command} [type] [term]. Supported types are ` + supportedTypes.map(x => "`" + x + "`").join(", ");
             message.channel.send(response);
```

Comment:
If you do any of the shorthand forms without arguments (such as `!item`), this will display the wrong error message.